tests/unit/layouts/ModelizerTextLayout.spec.js

```diff
@@ -173,7 +186,7 @@ describe('Test component: ModelizerTextLayout', () => {
   });
 
   describe('Test hook function: onMounted', () => {
-    it('should subscribe to SelectFileTabEvent', () => {
+    it('should subscribe to SelectFileTabEvent and retrieve parseErrors', () => {
```

Comment:
why update the scenario but no change within the test ?

Reply:
Nice catch, I initially added a test here, but removed it. I forgot to remove that part